apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -60,7 +63,15 @@ const {
     relatedManyToManyResolvers,
 } = require('@condo/domains/ticket/utils/serverSchema/TicketChange')
 const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
+
+
+const phoneWhiteList = Object.keys(conf.SMS_WHITE_LIST ? JSON.parse(conf.SMS_WHITE_LIST) : {})
+
+const redisGuard = new RedisGuard()
```

Comment:
could you please create one global redisGuard instance and use it everywhere. Because every new RedisGuard generate a new redis connection.